src/site/_filters/md-block.js

```diff
@@ -42,11 +42,8 @@ mdLib.renderer.rules = {...mdLib.renderer.rules, ...rules};
 /**
  * Render content as markdown.
  * @param {string} content
- * @return {?string}
+ * @return {string|undefined}
  */
 module.exports = (content) => {
-  if (!content) {
-    return;
-  }
-  return mdLib.render(content);
+  return content ? mdLib.render(content) : undefined;
```

Comment:
I think it's personal preference but I find the original version easier to read at a glance.
This is also another instance where I'm wondering if we have to explicitly return `undefined` or if we can rely on the implicit return value.

Reply:
For readability I can easily fix that, but the `undefined` issue from above still exists.

Reply:
For readability I can easily break it up, but for returning `undefined` that issue still stands.